rele/worker.py

```diff
@@ -144,6 +152,13 @@ def create_and_run(subs, config):
         config.credentials,
         config.ack_deadline,
         config.threads_per_subscription,
+        config.retry_policy,
+        {
+            "dead_letter_topic_id": config.dead_letter_topic_id,
+            "max_delivery_attempts": config.max_delivery_attempts,
+        }
+        if config.dead_letter_topic_id
+        else None,
```

Comment:
```suggestion
       config.dead_letter
```
Id suggest that the config should be handling the deadletter, and know all about how to set it up.